### PR TITLE
Multi-mission radioisotope thermoelectric generator knowledge

### PR DESCRIPTION
**Pull request by b4d**
https://github.com/instructlab/taxonomy/issues/839

### CODE REVIEW

**Review by bjhargrave — CHANGES_REQUESTED** [review]
The folder name has a misspelling that should be corrected.

`knowledge/science/astronomy_tech/radioisotope_thermo_generator/`

**Comment by jjasghar** [comment]
Please fix the linting issue(s).

File: knowledge/science/astronomy_tech/radioisotope_thermo_generator/qna.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+created_by: Deni B
```
Comment (jjasghar) [inline knowledge/science/astronomy_tech/radioisotope_thermo_generator/qna.yaml:1]: Please put your GitHub Username here

**Review by jjasghar — CHANGES_REQUESTED** [review]
Please resolve this issue.

**Comment by jjasghar** [comment]
Please delete the `.gitignore` and fix the DCO issue too.

**Comment by jjasghar** [comment]
Please rebase off of main, and only commit your `attribution.txt` and `qna.yaml`.

**Review by jjasghar — CHANGES_REQUESTED** [review]
Please look above about the rebase.

**Comment by jjasghar** [comment]
Closing due to combined PR here:  #1142

**Pull request closed without merging**